src/workspaces/components/VocabulariesTable.tsx

```diff
@@ -10,8 +10,8 @@ import { Box, styled } from '@material-ui/core'
 
 const columns: DataColumn<Vocabulary>[] = [
   {
-    title: t`uri`,
-    field: 'vocabulary',
+    title: t`label`,
+    field: 'label',
```

Comment:
What should be outputted if label is not filled in?

Reply:
Clearly we cannot have labels required yet. So I was thinking of two options.
1) adding something like "[Chybějící název]" when the label is missing
2) empty label 

For me 2) was sufficient for me **for now** as it did not require any implementation :). If you think we should have 1) or have other idea, let me know I will implement it this way.

Reply:
Fair enough :-) It's OK as it is.